README.md

```diff
@@ -44,7 +44,12 @@ f62cebbc2e43        herdbook_main              "/bin/sh -c /entrypo…"   3 hour
 
 ```
 
-To access the local server deployed open this url https://localhost:8443 in your browser. You will need to configure the browser to allow self-signed localhost certificates. In Chrome, this can be done by accessing this property from the browser: `chrome://flags/#allow-insecure-localhost` and setting its vale to Enabled. In Firefox, when loading the url you can click on Advanced and  add the exception suggested from the browser. You can also use http://localhost:8080.
+To access the local server deployed open this url https://localhost:8443 in your browser. You will need to 
+configure the browser to allow self-signed localhost certificates. In Chrome, this can be done by accessing 
+this property from the browser: `chrome://flags/#allow-insecure-localhost` and setting its vale to Enabled. 
+In Firefox, when loading the url you can click on Advanced and  add the exception suggested from the browser. 
+You can also use http://localhost:8080 to access the frontend. Note though that if you edit frontend files at runtime, 
+recompilation changes are only visible at the dev port: 8443.
```

Comment:
Hmm I get live hot reload updates on 8080 as well as 8443, so I don't think this readme change is correct.

Reply:
yes, it does now, wonder why the other day it did not

Reply:
I removed the comment in the README